### PR TITLE
Aswin file v1

### PR DESCRIPTION
**Pull request by Aswin31671**
Format changed

### CODE REVIEW

File: filehandling.c [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        if(fp == NULL){      
+		printf("error open filn in read mode");
+	}
+	else{
+		for(i = 0;i < n;i++){
+			fread(&emp[i],sizeof(struct employee),n,fp);
+			printf("Employee name:%s\tEmployee id:%d\n",emp[i].employeeName,emp[i].employeeId);
+		}
+
+		}
```
Comment (P-Prajwal) [inline filehandling.c:41]: why this extra bracket here?
```suggestion
```

File: filehandling.c [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	FILE *fp = NULL;
+	int n;
+	printf("enter number of employee:"); 
+	scanf("%d",&n);
+	struct employee emp[n];
+	int i;
+	for(i = 0;i < n;i++){ 
+		printf("Enter the employee details\n"); 
+	        printf("Name:");
+                scanf("%s",emp[i].employeeName);
```
Comment (tklogesh) [inline filehandling.c:18]: give proper space after comma

**Comment by tklogesh** [comment]
Give proper space after comma, semicolon.. wherever its necessary (for, while loop and so on...)

File: filehandling.c (hunk 1 above)
Comment (Aswin31671) [reply filehandling.c:41]: sorry, now I removed that bracket please check.

File: filehandling.c (hunk 2 above)
Comment (Aswin31671) [reply filehandling.c:18]: now I changed

**Pull request closed without merging**